Microsoft.Quantum.Canon/StatePreparation/StatePreparation.qs

```diff
@@ -0,0 +1,208 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+namespace Microsoft.Quantum.Canon
+{
+    open Microsoft.Quantum.Primitive;
+    open Microsoft.Quantum.Extensions.Math;
+
+    // This library returns operations that prepare a specified quantum state
+    // from the computational basis state $\ket{0...0}$.
+    
+    /// # Summary
+    /// Returns a unitary operation $U$ that prepares an arbitrary quantum 
+    /// state $\ket{\psi}$ with positive coefficients $\alpha_j\ge 0$ from 
+    /// the $n$-qubit computational basis state $\ket{0...0}$.
+    ///
+    /// $U\ket{0...0}=\ket{\psi}=\frac{\sum^{2^n-1}_{j=0}\alpha_j \ket{j}}{\sqrt{\sum^{2^n-1}_{j=0}|\alpha_j|^2}}$.
```

Comment:
This equation isn't a part of the surrounding text, such that it's not clear that it acts as a definition for _U_. Perhaps something like the following?

```
/// The action of U on a newly-allocated register is given by
/// \begin{align}
///     U \ket{0\cdots 0} = \ket{\psi} = \frac{\sum^{2^n-1}_{j=0}\alpha_j \ket{j}}{\sqrt{\sum^{2^n-1}_{j=0}|\alpha_j|^2}}$.
/// \end{align}
```

Reply:
Fixed